webapp/src/DOMJudgeBundle/Controller/Jury/RejudgingController.php

```diff
@@ -419,6 +420,82 @@ public function finishAction(Request $request, RejudgingService $rejudgingServic
         }
     }
 
+    /**
+     * @Route("/rejudging/add", name="jury_rejudging_add")
```

Comment:
Nitpick: the other endpoints are under `/rejudgings` (and we have that for other things likes teams and users as well), so I'd do that here too.